src/wokwi_client/client.py

```diff
@@ -233,3 +240,23 @@ async def set_control(
             value: Control value to set (float).
         """
         return await set_control(self._transport, part=part, control=control, value=value)
+
+    async def framebuffer_read(self, id: str) -> ResponseMessage:
```

Comment:
What's the use case for this API? 

IMHO if there's no good use case, let's not expose it, since it is can be confusing when you have both framebuffer_read and framebuffer_png_bytes.